### PR TITLE
[UCHAT-4597] Added prometheus metric collection for mobile events.

### PR DESCRIPTION
**Pull request by amitmall**

### CODE REVIEW

File: model/mobile_metric.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	"io"
+	"time"
+)
+
+type DeviceDetails struct {
+	Country        string `json:"country"`
+	APILevel       string `json:"api_level"`
+	BuildNumber    string `json:"build_number"`
+	BundleId       string `json:"bundle_id"`
+	DeviceId       string `json:"device_id"`
```
Comment (bhaskaruniz) [inline model/mobile_metric.go:14]: Is there any field to identify whether its Android or IOS device?

File: utils/prometheus_metrics.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+func NewClientMetrics() *ClientMetrics {
+	histogramMetrics := prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "mobile_metrics_seconds",
+			Help:    "Mobile metrics",
+			Buckets: prometheus.ExponentialBuckets(0.01, 1.3, 36),
+		},
+		[]string{"event", "country", "app_version", "server_version"},
+	)
+	prometheus.MustRegister(histogramMetrics)
```
Comment (bhaskaruniz) [inline utils/prometheus_metrics.go:27]: what happens for failure cases where call was not successful

File: model/mobile_metric.go (hunk 1 above)
Comment (amitmall) [reply model/mobile_metric.go:14]: Currently we dont have. But with model name we can differentiate.

File: utils/prometheus_metrics.go (hunk 2 above)
Comment (amitmall) [reply utils/prometheus_metrics.go:27]: MustRegister doesnt return any error. I am handling panic case using recover.

**Pull request merged**